packages/react/popover/src/Popover.tsx

```diff
@@ -40,10 +41,11 @@ type PopoverOwnProps = {
   open?: boolean;
   defaultOpen?: boolean;
   onOpenChange?: (open: boolean) => void;
+  modal?: boolean;
 };
 
 const Popover: React.FC<PopoverOwnProps> = (props) => {
-  const { children, open: openProp, defaultOpen, onOpenChange } = props;
+  const { children, open: openProp, defaultOpen, onOpenChange, modal = true } = props;
```

Comment:
Is defaulting to `modal` the expected here? I'm in two minds as whether it should be the other way around.

Reply:
Yeah I also feel like `modal={false}` is a better default for `Popover`. How do we feel about having different defaults for different primitives?

> Note: that was already the case but with the granular props instead.

Reply:
I agree, I think it should default to `false`. I'm fine with differing defaults, I think we should optimise for the common case and it (kind of) was the original behaviour. This does make the closing toggle issue much more obvious though.